analyses/interaction-plots/scripts/02-process_mutations.R

```diff
@@ -133,7 +133,7 @@ option_list <- list(
   make_option(
     opt_str = "--vaf",
     type = "numeric",
-    default = 0.2,
+    default = 0.05,
```

Comment:
Is there a particular reason you are changing this default value?

Reply:
Not really, I guess I just thought it was too strict before.